### PR TITLE
Update to CONTENT IDEA REQUEST 120542

### PR DESCRIPTION
**Pull request by yanqingw-amy**
update "To specify that the WDAC policy..." this sentence

### CODE REVIEW

File: windows/security/threat-protection/windows-defender-application-control/create-initial-default-policy.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ To create a WDAC policy, copy each of the following commands into an elevated Wi
    > - You can add the **-MultiplePolicyFormat** parameter when creating policies which will be deployed to computers which are running Windows build 1903+. For more information about multiple policies, see [Deploy multiple Windows Defender Application Control policies](deploy-multiple-windows-defender-application-control-policies.md).
    > - You can add the **-Fallback** parameter to catch any applications not discovered using the primary file rule level specified by the **-Level** parameter. For more information about file rule level options, see [Windows Defender Application Control file rule levels](select-types-of-rules-to-create.md).
    > 
-   > - To specify that the WDAC policy scan only a specific drive, include the **-ScanPath** parameter followed by a path. Without this parameter, the entire system is scanned.
+   > - To specify that the WDAC policy scan only a specific drive, include the **-ScanPath** parameter followed by a path. Without this parameter, the tool will scan the drive C in default.
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-application-control/create-initial-default-policy.md:73]: Minor grammar detail:
```suggestion
   > - To specify that the WDAC policy scan only a specific drive, include the **-ScanPath** parameter followed by a path. Without this parameter, the tool will scan the C-drive by default.
```

Comment (yanqingw-amy) [reply windows/security/threat-protection/windows-defender-application-control/create-initial-default-policy.md]: @illfated thanks for your comments, updated now! could you please help merge it?

**Comment by illfated** [comment]
Thank you for adding my suggested change.
Please note, I am not affiliated with Microsoft, or the Microsoft Docs teams here on GitHub.
That being said, I will add a comment to aid you getting a follow-up from the MS Docs team.

**Comment by illfated** [comment]
#sign-off

**Pull request merged**